pkg/controller/knativeserving/knativeserving_controller.go

```diff
@@ -219,6 +232,21 @@ func (r *ReconcileKnativeServing) install(instance *servingv1alpha1.KnativeServi
 	return nil
 }
 
+func (r *ReconcileKnativeServing) delete(instance *servingv1alpha1.KnativeServing) error {
```

Comment:
Don't we still need the `isInteresting` query in here somewhere?

Reply:
Nope, we only enter here after we already checked it above.

Reply:
pb